qa/tasks/cephfs/test_multifs_auth.py

```diff
@@ -76,16 +76,20 @@ def setUp(self):
     def test_rw_with_fsname_and_no_path_in_cap(self):
         PERM = 'rw'
         self.captester.write_test_files(self.mounts)
-        keyring_paths = self._create_client(PERM, fsname=True)
-        self.remount_with_new_client(keyring_paths)
+
+        moncap, osdcap, mdscap = self._gen_caps(PERM, both_fsnames=True)
+        keyring = self.create_client(self.client_id, moncap, osdcap, mdscap)
```

Comment:
If `create_client` is only called from this source, its a tiny bit better to pass in a set (or keyworded args) of caps and rename it as `_create_client` to signify that it's (kind of) and internal routine.

Reply:
It's not an internal routine, it's a helper method in CephFSTestCase.